packages/web-components/tests/e2e-storybook/cypress/integration/carousel/carousel.e2e.js

```diff
@@ -0,0 +1,343 @@
+/**
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+/**
+ * Sets the correct path
+ *
+ * @param default - Path to default variant
+ * @param withImages - path to variant with images
+ * @param withCta - path to variant with videos
+ * @param withCta - path to variant with both images and videos
```

Comment:
Good work documenting the code, but lets make sure the @param names match the keys in the object